src/Dots/Dots.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component, PropTypes } from 'react'
+
+import Dot from './Dot'
+import dynStyle from './style'
+
+const maxDots = 3
+const maxOpacity = 0.6
+const minOpacity = 0.1
+const timeout = 1000
+
+class Dots extends Component {
+  constructor(props, context) {
+    super(props, context)
+
+    // FIXME : google help me to initialize this
+    const opacities = []
+    for (let i = 0; i < maxDots; ++i) opacities.push(minOpacity)
```

Comment:
I'll stick with my version for the moment.
I would prefer not to import lodash in this HOC.